jenkins/ansible/roles/createNightlyBuildJob/files/utils.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python3
+#
+# Authors:
+#     Dinesh Prasanth M K <dmoluguw@redhat.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright (C) 2017 Red Hat, Inc.
+# All rights reserved.
+#
+
+import requests
+from requests.auth import HTTPBasicAuth
+import datetime
+
+class Utils:
+
+    DELETE_RESULT = {
+            204: "was removed successfully",
+            400: "can't be deleted currently",
+            403: "authroization failed",
+            404: "doesn't exists"
+    }
+
+    def __init__(self, repo, copr_url=None):
```

Comment:
The 'repo' parameter should not be needed here since the repo name will be provided to getProjectID().

Reply:
The constructor can take a 'client' parameter which is the COPR API client object. If possible, the copr_url should be obtained from the client parameter.